projects/ngx-d3timeline/src/lib/ngx-d3timeline.component.ts

```diff
@@ -62,6 +60,8 @@ export class NgxD3timelineComponent
   }
 
   @Output() activityDropped = new EventEmitter<Activity>();
+  @Output() hovered = new EventEmitter<Activity>();
+  @Output() unhovered = new EventEmitter<Activity>();
```

Comment:
```suggestion
  @Output() activityUnhovered = new EventEmitter<Activity>();
```

Reply:
This event will also have to fire when a resource is hovered